styles/tailwind.css

```diff
@@ -27,7 +27,7 @@
 
 @layer utilities {
   .focus-ring {
-    @apply focus:outline-none focus-within:outline-none transition disabled:ring-0 hover:ring-2 focus:ring-2 focus-within:ring-2 group-hover:ring-2 hover:ring-team-current focus:ring-team-current focus-within:ring-team-current group-hover:ring-team-current ring-team-current ring-offset-4 dark:ring-offset-gray-900 ring-offset-white;
+    @apply focus:outline-none focus-within:outline-none transition duration-300 disabled:ring-0 hover:ring-2 focus:ring-2 focus-within:ring-2 group-hover:ring-2 group-focus:ring-2 hover:ring-team-current focus:ring-team-current focus-within:ring-team-current group-hover:ring-team-current group-focus:ring-team-current ring-team-current ring-offset-4 dark:ring-offset-gray-900 ring-offset-white;
```

Comment:
changed the duration of the ring transition to match with framer-motion. This looks better when combining arrow and ring animations in a single element (like the featured card).